docs/reference/expectations/result_format.md

```diff
@@ -4,66 +4,62 @@ title: Result format
 
 
 The `result_format` parameter may be either a string or a dictionary which specifies the fields to return in `result`.
-  * For string usage, see `result_format` values.
+  * For string usage, see [`result_format` values and fields](#result_format-values-and-fields).
   * For dictionary usage, `result_format` which may include the following keys:
     * `result_format`: Sets the fields to return in result.
+    * `unexpected_index_column_names`: Defines the primary key (PK) columns used to represent unexpected results.
+    * `return_unexpected_index_query`: When running validations, a query (or a set of indices) will be returned that will
+      allow you to retrieve the full set of unexpected results defined by the PK.  Setting this value to `False` will 
+      suppress the output (default is `True`).
     * `partial_unexpected_count`: Sets the number of results to include in partial_unexpected_count, if applicable. If 
       set to 0, this will suppress the unexpected counts.
     * `include_unexpected_rows`: When running validations, this will return the entire row for each unexpected value in
       dictionary form. When using `include_unexpected_rows`, you must explicitly specify `result_format` as well, and
       `result_format` must be more verbose than `BOOLEAN_ONLY`. *WARNING: *
-
+  
   :::warning
   `include_unexpected_rows` returns EVERY row for each unexpected value; for large tables, this could return an 
   unwieldy amount of data.
   :::
 
 ## Configure Result Format
-Result Format can be applied to either a single Expectation or an entire Checkpoint.
+Result Format can be applied to either a single Expectation or an entire Checkpoint. When configured at the Expectation-level, 
+the configuration will not be persisted, and you will receive a `UserWarning`. We therefore recommend that the Expectation-level
+configuration be used for exploratory analysis, with the final configuration added at the Checkpoint-level.
+
 ### Expectation Level Config
-To apply `result_format` to an Expectation, pass it into the Expectation's configuration:
-```python
-# first obtain a validator object, for instance by running the `$ great_expectations suite new` notebook.
-validation_result = validator.expect_column_values_to_be_between(
-    column="pickup_location_id",
-    min_value=0,
-    max_value=100,
-    result_format="COMPLETE",
-    include_unexpected_rows=True
-)
-unexpected_index_list = validation_result["result"]["unexpected_index_list"]
-unexpected_list = validation_result["result"]["unexpected_list"]
+To apply `result_format` to an Expectation, pass it into the Expectation. We will first need to obtain a Validator object instance by running the `$ great_expectations suite new` command.
```

Comment:
Nit: Just set off the CLI in a parenthetical since there are other ways to set up a validator (the CLI just generating the boilerplate).
```suggestion
To apply `result_format` to an Expectation, pass it into the Expectation. We will first need to obtain a Validator object instance (e.g. by running the `$ great_expectations suite new` command).
```